### PR TITLE
Announcing machine to machine authorization

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

File: _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- 2016-09-28-announcing-identity-glossary
+- 2016-08-25-announcing-Auth0-Guardian-a-new-way-to-login
+- 2016-08-24-announcing-password-breach-detection
+---
+
+Today, we are introducing a new authorization feature for all of our users; [Machine to Machine Authorization](https://auth0.com/docs/api-auth/grant/client-credentials). Read on to find out how it works and how to use it.
+
+{% include tweet_quote.html quote_text="Introducing a new authorization feature for all Auth0 users: Machine to Machine Authorization!" %}
+
+A non-interactive application (a CLI, a daemon, or a service running on your backend), can now directly ask Auth0 for an `access_token` by using its client credentials, **Client ID** and **Client Secret** to authenticate.
```
Comment (kmaida) [inline _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown]: Remove this comma: ` your backend), can`

Add a comma after `**Client Secret**`.

File: _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+![Client Credentials Grant](https://cdn.auth0.com/docs/media/articles/api-auth/client-credentials-grant.png)
+_Client Credentials grant_
+
+Once the `access_token` has been returned, the application can use the `access_token` to call an API and have access to its resource on behalf of itself.
```
Comment (kmaida) [inline _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown]: This sentence is somewhat confusing, mainly this part: `have access to its resource on behalf of itself`

I'm not sure I'm entirely following the meaning of this. Can it be clarified?

File: _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## How do I make this work?
```
Comment (kmaida) [inline _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown]: Change to `How Do I Make This Work?` to be consistent with the capitalization of the other header.

File: _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+* An Authorization Server (Auth0)
+* A Resource Server - **World Mappers API**
+* A client that consumes the API - **Gift Deliveries**
+
+Clone the project from [Github](https://github.com/auth0-samples/auth0-api-auth-samples/tree/master/machine-to-machine), set it up on your computer and experience the awesomeness of Machine to Machine API Authorization.
+
+## Conclusion
+
+Auth0 now offers APIs as first class citizens. Auth0 account owners can open up their APIs to the world while security is enforced by user consent. Apart from machine to machine authorization, Auth0 offers other ways to perform API authorization. You can learn how to perform these forms of API authorization by going through our clearly laid out [documentation](https://auth0.com/docs/api-auth).
```
Comment (kmaida) [inline _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown]: I'd recommend removing `clearly laid out` from the last sentence and just leave it as "through our documentation".

File: _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown (hunk 2 above)
Comment (unicodeveloper) [inline _posts/2017-03-20-announcing-machine-to-machine-authorization.markdown]: I mean an API resource. I'll just write that

**Comment by kmaida** [comment]
June 7, 2018 UPDATE:

We would like to roll out this blog post to complement the launch of the Machine to Machine landing page. We should update it to incorporate some use cases for the M2M API.

**Comment by sebadoom** [comment]
Superseded by #1563

**Pull request closed without merging**